cluster/docker/config-default.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Contains configuration values for the bare-metal cluster
+
+# (0) Versions
+export ETCD_VERSION=${ETCD_VERSION:-"2.0.12"}
```

Comment:
etcd has been bumped to 2.2.1 via #16694, yet something might worth digging #17205. I'm not sure it's ubuntu-exclusive (required flag missed issue) or not though.